### ISSUE TITLE
Improve accessibility of messages

### ISSUE THREAD

**Issue by rugk** [rugk/offline-qr-code#125]
Problem: When you do not see the icons/style of the messages, you cannot differentiate whether these are errors, info messages, or so…

Solution (at least what I'd say is the easiest/best): Add `aria-label` to the message icon (or box?).

Of course, the static HTML needs to be adjusted.
In the `MessageHandler` you need to adjust the `setMessageDesign` and the corresponding tests, to also check this attribute.
